cmake/modules/RootBuildOptions.cmake

```diff
@@ -121,6 +121,7 @@ ROOT_BUILD_OPTION(cudnn ON "Enable support for cuDNN (default when Cuda is enabl
 ROOT_BUILD_OPTION(cxxmodules OFF "Enable support for C++ modules (deprecated)")
 ROOT_BUILD_OPTION(daos OFF "Enable RNTuple support for Intel DAOS")
 ROOT_BUILD_OPTION(dataframe ON "Enable ROOT RDataFrame")
+ROOT_BUILD_OPTION(docu OFF "Build the HTML based ROOT documentation (requires Doxygen, and does not work on Windows)")
```

Comment:
```suggestion
ROOT_BUILD_OPTION(documentation OFF "Build the HTML based ROOT documentation (requires Doxygen, and does not work on Windows)")
```

Reply:
Hi Philippe! Can you review  #15160 instead, since this draft PR is working on top of the changes in that PR?